### PR TITLE
Fixing annotations to remove build warnings

### PR DESCRIPTION
**Pull request by avgAGB**
There are many areas in which this would need to be done, so it might make sense to make the changes in waves through various modules/packages.

### CODE REVIEW

**Review by ivakegg — DISMISSED** [review]

File: warehouse/core/src/main/java/datawave/ingest/mapreduce/handler/dateindex/DateIndexUtil.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public class DateIndexUtil {
      * Format the date into yyyyMMdd
      * 
      * @param date
+     *            then date to be formatted
```
Comment (apmoriarty) [inline warehouse/core/src/main/java/datawave/ingest/mapreduce/handler/dateindex/DateIndexUtil.java:24]: the date

**Review by apmoriarty — DISMISSED** [review]
lgtm -- noticed one typo

File: warehouse/core/src/main/java/datawave/ingest/mapreduce/handler/dateindex/DateIndexUtil.java (hunk 1 above)
Comment (avgAGB) [reply warehouse/core/src/main/java/datawave/ingest/mapreduce/handler/dateindex/DateIndexUtil.java:24]: thanks. Still have a few more annotations to go but I'll make this adjustment as well.

**Review by apmoriarty — DISMISSED** [review]

**Comment by avgAGB** [comment]
Circling back on this : I've still got quite a bit of annotations to fix just in the warehouse module, so I'm going to wrap this section up and break up the next changes by package to make it a little more digestible.

**Review by ivakegg — APPROVED** [review]

**Review by alerman — APPROVED** [review]

**Pull request merged**